std/dispatch.solc

```diff
@@ -0,0 +1,293 @@
+import std;
+
+pragma no-patterson-condition RunDispatch, ExecMethod;
+
+// --- Preliminaries ---
+
+data Bool = True | False;
```

Comment:
Using the builtin `bool` type gave me a bunch of specializer errors I didn't understand. I'll try to find a minimal reproducer and submit a bug.